### PR TITLE
5 get users

### PR DESCRIPTION
**Pull request by althrntn**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -93,4 +93,12 @@ describe.only("GET /api/users", () => {
         });
       });
   });
+  test("404: returns a not found message when incorrect path enterred", () => {
```
Comment (popatre) [inline __tests__/app.test.js:96]: Spelling on entered.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -93,4 +93,12 @@ describe.only("GET /api/users", () => {
...
+    return request(app)
+      .get("/api/usr")
```
Comment (popatre) [inline __tests__/app.test.js:98]: If you've already done a bad route test elsewhere, you possibly don't need another, as any no existent route will be caught. 
If this is the first time you're testing a bad route, then you'd need this.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -77,3 +77,20 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/users", () => {
+  test("200: returns an array of user objects, each with username, name and avatar_url properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((res) => {
```
Comment (popatre) [inline __tests__/app.test.js:86]: - [ ] Response needs to be sent back as an object with the key of users
- [ ] Need to check the length of the array to check you're getting all the users

File: controllers/user-controllers.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const { fetchUsers } = require("../models/user-models");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    console.log(users);
```
Comment (popatre) [inline controllers/user-controllers.js]: Remove console log please

File: controllers/user-controllers.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    res.status(200).send(users);
```
Comment (popatre) [inline controllers/user-controllers.js:5]: Need to send the response as an object with a key of users

File: db-check.txt [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
...                     | topic |    author     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                body                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |     created_at      | votes
```
Comment (popatre) [inline db-check.txt:2]: Probably dont need this file as part of the PR

Comment (popatre) [reply db-check.txt:2]: Could set up .gitignore to ignore these

**Review by popatre — CHANGES_REQUESTED** [review]
Just some small changes please

File: __tests__/app.test.js (hunk 1 above)
Comment (althrntn) [reply __tests__/app.test.js]: test removed

File: __tests__/app.test.js (hunk 2 above)
Comment (althrntn) [reply __tests__/app.test.js]: test removed

File: __tests__/app.test.js (hunk 3 above)
Comment (althrntn) [reply __tests__/app.test.js:86]: test altered

File: controllers/user-controllers.js (hunk 4 above)
Comment (althrntn) [reply controllers/user-controllers.js]: wasn't in my version. Already gone

File: controllers/user-controllers.js (hunk 5 above)
Comment (althrntn) [reply controllers/user-controllers.js]: controller altered

File: db-check.txt (hunk 6 above)
Comment (althrntn) [reply db-check.txt:2]: added to .gitignore

**Review by popatre — APPROVED** [review]
Looks good to go!

**Pull request merged**